docs/using-the-snowplow-console/managing-console-api-authentication/index.md

```diff
@@ -55,3 +53,46 @@ You may then use this access token value to supply authorization headers for sub
 ```bash
 curl --header 'Authorization: Bearer <JWT>'
 ```
+
+Authenticating with v2 only required the API key secret. While this method and the respective credentials will continue to work, we strongly advise you to use v3 detailed below.
+
+## Credentials UI v3
+
+The following view is available for all customers under [BDP Console settings](https://console.snowplowanalytics.com/credentials):
+
+![](images/accessing-generated-api-keys.png)
+
+API keys generation view
+
+It is possible to have multiple different keys, and deletion of a key is also possible. When a new API key is created, the following view shows up:
+
+![](images/generated-api-key-v3.png)
+
+Showing a newly created API key
+
+With v3 you will need both the API key ID and the API key secret. The secret is equivalent to a username and a password combined, and should be handled as such at all times. As soon as you have an API key, it is straightforward to exchange it for a JWT. If you would do that using curl, it would look as follows:
```

Comment:
In v3, both the API key ID and API key secret are required. The secret functions like a combination of a username and password, and should be treated with the same level of security. Once you have an API key, exchanging it for a JWT is straightforward. For example, using curl, the process would look like this: